### DISCUSSION TITLE
DAVX5 Outdated

### DISCUSSION THREAD

**Discussion by DickinsonSa** [TechbeeAT/jtxBoard#328] [Q&A] [answered]
I'm getting a sync message asking me to upgrade to 4.3.1 (or later) of DAVx5 - synchronisation is currently disabled.

My current version is 4.3-gplay.

Is this a bug or something that I can address?

**Comment by patrickunterwegs** [comment, marked as answer]
Hello @DickinsonSa , 
the message is there to tell you what you should do. If you're in the beta-program of jtx Board on the play store, you should also join the DAVx5 beta-program there to get the latest version!

**Reply by DickinsonSa** [reply]
Thanks Patrick, I wasn't in both betas. I'll go join now.
